hw3/css/stylehw3.css

```diff
@@ -0,0 +1,477 @@
+* {
+	box-sizing: border-box;
+}
+body{
+	font: 14px/26px Arial, sans-serif;
+}
+/*HEADER*/
+.header {
+	background-image: url(../img/header.png);
+	background-repeat: repeat-x;
+	box-shadow: 0px 1px 13px 0px rgba(0, 0, 0, 0.294);
+	position: relative;
+}
+.header__wrapper {
+	width: 80%;
+	margin: 0 auto;
+	padding: 14px 0 15px;
+	line-height: 39px;
+	color: #fff;
+	display: flex;
+	justify-content: space-between;
+	position: relative;
+}
+.header__logo {
+	font-size: 48px;
+	font-family: Dunkirk, sans-serif;
+	display: flex;
+	align-items: center;
+}
+.header__logo::before {
+	content: url(../img/logo.png);
+	display: block;
+} 
+.header__menu {
+	display: flex;
+}
+.header__menu__item{
+	margin-left: 25px;
+}
+.header__menu__link {
+	color: #ffffff;
+	font-size: 18px;
+	display: block;
+}
+.header__menu__link:hover {
+	color: #4147e8;
+}
+/*RESPONSIVE MENU*/
+.responsive-menu {
+	position: relative;
+	font-size: 40px;
+	padding-left: 15px;
+	display: none;
+	cursor: pointer;
+}
+.dropdown-menu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	background-color: #2b2626;
+	z-index: 200;
```

Comment:
use reasonable values here, if you need 1 or 2 use it. As far as you will have popups hints and so on it will be hell to check all z-indexes in app